pkgs/by-name/dy/dynamodb-local/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchurl
+, jre
+, makeBinaryWrapper
+}:
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
I'm wondering if we can use `stdenNoCC` ?

Reply:
tested it works well